sonar-project.properties

```
@@ -3,5 +3,6 @@ sonar.organization=moveit
 
 sonar.python.version=3
 sonar.sources=.work/target_ws/src
+sonar.cfamily.llvm-cov.reportPath=.work/target_ws/coverage.info
```

It doesn't seem to have picked up any coverage info. Maybe this has to be a directory?

```suggestion
sonar.cfamily.llvm-cov.reportPath=.work/target_ws
```

---

actually nope (hiding my above incorrect comment) I just looked at the [example for this](https://github.com/sonarsource-cfamily-examples/linux-cmake-llvm-cov-gh-actions-sc/blob/main/.github/workflows/build.yml#L48) and it is a file path, not a directory path.